### ISSUE TITLE
Add icon to windows executable file

### ISSUE THREAD

**Issue by Tarek-Hasan** [stuntrally/stuntrally3#26]
First, thanks for the new release. v3.1 looks better and runs smoothly compared to v2.7.

As I mentioned in the [original issue](https://github.com/stuntrally/stuntrally/issues/77) of old StuntRally repo - in windows without integrated icons, stuntrally exe and shortcut files looks like corrupted or missing.

Hope, you'll fix this soon.

**Comment by cryham** [comment]
Yeah I know. Icons are also missing for Linux binaries and in Windows installer too. I didn't care much about those. Will add to roadmap and try doing someday.

**Comment by cryham** [comment]
Ok I did this with SR 3.2. It's a manual step, more to do each time.
I did a quick look and I don't think it can be added to CMake, to be automatic.
Would be way better.

**Comment by Tarek-Hasan** [comment]
Thanks for the new release and solving the issue.

But there's a problem with uninstalling from windows settings. It searches for uninst.exe in \Stunt Rally 3.2\sr folder, but the installer puts the uninst.exe file in \Stunt Rally 3.2 which results in an error.
![Screenshot 2024-09-24 143518](https://github.com/user-attachments/assets/de765e4e-a1d5-4afc-9e22-c89577fdd6da)
